### PR TITLE
New Package py-portend

### PR DESCRIPTION
**Pull request by marcmengel**
tcp port monitoring and discovery package, dependency of cherrypy framework

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-portend/package.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from spack import *
+
+
+class PyPortend(PythonPackage):
+    """TCP port monitoring and discovery """
+
+    homepage = "https://github.com/jaraco/portend"
+    url      = "https://files.pythonhosted.org/packages/2c/59/948666fc2455ae471efd40cb2a9a990f5f6f2354a9a6b228e29b9fb4a307/portend-2.5.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-portend/package.py]:
```suggestion
    url      = "https://pypi.io/packages/source/p/portend/portend-2.5.tar.gz"
```

File: var/spack/repos/builtin/packages/py-portend/package.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+    version('2.5', sha256='19dc27bfb3c72471bd30a235a4d5fbefef8a7e31cab367744b5d87a205e7bfd9')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-portend/package.py]: Missing dependencies:
```
python_requires = >=2.7
install_requires =
        tempora>=1.8
setup_requires = setuptools_scm >= 1.15.0
```

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-portend/package.py]: Still missing these dependencies

Comment (marcmengel) [reply var/spack/repos/builtin/packages/py-portend/package.py]: Sorry, thought I did that, must have dreamt it

**Pull request merged**